sw/device/silicon_creator/lib/error.h

```diff
@@ -47,6 +48,7 @@ enum module_ {
   X(kErrorUartBadBaudRate,          ERROR_(2, kModuleUart, kInvalidArgument)), \
   X(kErrorHmacInvalidArgument,      ERROR_(1, kModuleHmac, kInvalidArgument)), \
   X(kErrorSigverifyInvalidArgument, ERROR_(1, kModuleSigverify, kInvalidArgument)), \
+  X(kErrorOtpBusy,             ERROR_(1, kModuleOtp,  kUnavailable)),  \
```

Comment:
nit: This section of code has clang-format turned off.  Please horizontally align your ERROR_ with the rest.